src/evaluation_system/tests/esgf_test.py

```diff
@@ -122,15 +123,15 @@ def test_freva_esgf_method(dummy_config):
     result_to_be = {
         "version": "20190219",
         "activity_id": ["ScenarioMIP"],
-        "master_id": "CMIP6.ScenarioMIP.CNRM-CERFACS.CNRM-CM6-1.ssp585.r1i1p1f2.E3hr.vas.gr",
+        "master_id": "CMIP6.ScenarioMIP.CNRM-CERFACS.CNRM-CM6-1.ssp585.r1i1p1f2.E3hr.was.gr",
         "mip_era": ["CMIP6"],
         "product": ["model-output"],
         "source_id": ["CNRM-CM6-1"],
         "url": [
             "http://esgf3.dkrz.de/thredds/catalog/esgcet/1586/CMIP6.ScenarioMIP.CNRM-CERFACS.CNRM-CM6-1.ssp585.r1i1p1f2.E3hr.vas.gr.v20190219.xml#CMIP6.ScenarioMIP.CNRM-CERFACS.CNRM-CM6-1.ssp585.r1i1p1f2.E3hr.vas.gr.v20190219|application/xml+thredds|THREDDS",
             "http://esgf3.dkrz.de/las/getUI.do?catid=649F6BBCDD98B0633CF0042D119CC69D_ns_CMIP6.ScenarioMIP.CNRM-CERFACS.CNRM-CM6-1.ssp585.r1i1p1f2.E3hr.vas.gr.v20190219|application/las|LAS",
         ],
-        "variable": ["vas"],
+        "variable": ["was"],
```

Comment:
vas stands for "v" (north-south) component of atmospheric surface wind. was would be "w" (up-down) component of atmospheric surface wind.

Reply:
right, this change has to be rejected